src/Lean/Message.lean

```diff
@@ -62,8 +62,8 @@ structure TraceData where
 inductive MessageData where
   /-- Eagerly formatted text. We inspect this in various hacks, so it is not immediately subsumed by `ofPPFormat`. -/
   | ofFormat          : Format → MessageData
-  /-- Lazily formatted text. -/
-  | ofPPFormat        : PPFormat → MessageData
+  /-- Formatted text with info annotations -/
+  | ofFormatWithInfos : FormatWithInfos → MessageData
```

Comment:
It is plausible that the comment on `ofFormat` no longer applies, and `ofFormat` now *is* subsumed, i.e., all the `MessageData`-inspecting functions could as well inspect `ofFormatWithInfos` instead.